nautobot/core/settings.py

```diff
@@ -530,8 +530,8 @@
 CELERY_TASK_TIME_LIMIT = int(os.getenv("NAUTOBOT_CELERY_TASK_TIME_LIMIT", 30 * 60))
 
 # The Redis connection defined in the CACHES config above for the broker and results backend
-CELERY_BROKER_URL = CACHES["default"]["LOCATION"]
-CELERY_RESULT_BACKEND = CACHES["default"]["LOCATION"]
+CELERY_BROKER_URL = os.getenv("NAUTOBOT_CELERY_BROKER_URL", CACHES["default"]["LOCATION"])
```

Comment:
Nearly all other variables in settings.py do not default to the env vars, that happens in the nautobot_config.py.j2.  We should be consistent.  However, I actually REALLY like the pattern of moving the default from env vars to settings.py and leaving nautobot_config.py only for values that need to be overridden.

Reply:
That's the direction we're moving. Right now it's only for the things that have been changed recently. We did it w/ the database settings and now Celery.

Reply:
Is there an issue to track that?  I'd be tempted to work it if you can define a bit the direction you want to go.  I can see this having major benefits simplifying several of my deployments.

Reply:
No not really just something we had verbally agreed upon.